sys/log/full/src/log.c

```diff
@@ -569,6 +599,21 @@ log_append_mbuf_typed_no_free(struct log *log, uint8_t module, uint8_t level,
         goto err;
     }
 
+    /*
+     * Check that the log body length is less than the maximum entry. This code
+     * may appear a bit odd in that it checks that the length is greater than
+     * a log entry header length. The reason for this check is to ensure any
+     * error handling of this case to be the same as it was before the
+     * maximum entry length was checked.
+     */
+    len = os_mbuf_len(om);
+    if (len > LOG_ENTRY_HDR_SIZE) {
+        rc = log_chk_max_entry_len(log, len - LOG_ENTRY_HDR_SIZE);
+        if (rc) {
```

Comment:
nitpick: once `if (rc )` is used, other time ` if(rc != OS_OK)` ;)

Reply:
Fixed and pushed.